### ISSUE TITLE
Error in cgat fasta2bed

### ISSUE THREAD

**Issue by biozzq** [cgat-developers/cgat-apps#78]
Hi @Acribbs 

I found that the output of `fasta2bed` always missed the first ungapped segment. For example;
```
cat test.fa 
>1
ATGCNATG
cat test.fa | cgat fasta2bed -m ungapped
1	5	8	1	0.0000
```
Best wishes,
Zheng zhuqing

**Comment by Acribbs** [comment]
thanks for reporting this. Things were extremely busy so I missed this. I will take a look and see if I can isolate the code

**Comment by Acribbs** [comment]
fixed with #81
